members/state_nh_gov_chris_sununu.yaml

```diff
@@ -278,7 +278,7 @@ contact_form:
           Wolfeboro: Wolfeboro
           Woodstock: Woodstock
       - name: County
-        selector: select[name='j_id0:j_id2:j_id10:j_id29'][class='oCounty']
+        selector: select[name*='County'][class='oCounty']
```

Comment:
Minor: Using `[class='oCounty']` requires that the entire class value be equal to `oCounty` vs if you did `select.oCounty` it matches if the element just includes that class. So in this case you could do `select.oCounty[name*='County']` for a `select` with the css class `oCounty` and a `name` attribute that contains `County`. 🤷‍♂ Probably fine as-is, but typically using `[class=...]` isn't desired

Reply:
Noted, good point. As you said, this is probably minor for this yaml, but will def keep it in mind for future changes. Gracias